crypto-hash-example/flb.conf

```diff
@@ -0,0 +1,20 @@
+[SERVICE]
+    flush      0.1
+    grace      2
+    log_level  info
+
+[INPUT]
+    name tail
+    read_from_head true
+    exit_on_eof true
```

Comment:
I'd add a comment indicating this may not flush all output in case people see occasional dropped output.